jsonstore/jsonstore.go

```diff
@@ -74,6 +78,14 @@ type UserCommentVote struct {
 	CommentID bson.ObjectId `bson:"commentID" json:"commentID"`
 }
 
+// Validator
+type Validator struct {
+	ID        bson.ObjectId   `bson:"_id" json:"_id"`
+	Name      string          `bson:"name" json:"name"`
+	PublicKey []byte          `bson:"publicKey" json:"publicKey"`
+	Votes     []bson.ObjectId `bson:"votes" json:"votes"`
```

Comment:
Validator documents can quickly grow in size (MongoDB documents have a 16MB size limit) if we store the ObjectIDs of voters in the same document (`Votes` array). There is no upper limit to the number of voters. I would suggest storing the `votes` count in the document itself and move the ObjectIDs of voters to a separate junction collection. This will also help us sort the validators efficiently.

Reply:
as discussed added UserValidatorVote collection